action.yml

```diff
@@ -7,7 +7,7 @@ branding:
 inputs:
   version:
     description: 'The version of buf to setup.'
-    default: 'latest'
+    default: '0.56.0'
```

Comment:
So the approach here would be "maxing out the latest" to `0.56.0`, for example, right? E.g. without upgrading the setup action, you're basically restricted to this maximum version, but users can explicitly set the version input to `latest` to go beyond this version, right? I am okay with this user experience, so I am going to stamp this, but we may want to call out setting `latest` explicitly as a supported behaviour in the README.

Reply:
Yea I agree that we should update the `README.md` with the ability to set `latest`. I'll include a note here.

Reply:
Also to be clear, users aren't "maxed out" by the setting - they can still explicitly set a version that's higher than `0.56.0`.